projects/packages/jetpack-mu-wpcom/src/features/launch-button/style.css

```diff
@@ -0,0 +1,3 @@
+#wpadminbar .launch-site {
+	background: var(--wp-admin-theme-color);
```

Comment:
I've used the admin color CSS variable to try to stay consistent with the admin bar but I noticed that these variables don't contain the right values. I think this is out of scope of the current PR. I know some folks are working on making sure the right colors are used everywhere.